src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -261,5 +263,15 @@ internal static ScaleVote GetAggregateScaleVote(IEnumerable<ScaleVote> votes, Sc
                 scaleMonitorsToSample.AddRange(scaleMonitors);
             }
         }
+
+        private string GetTargetScalerFunctionUniqueId(ITargetScaler scaler)
+        {
+            return $"{scaler.GetType().Assembly.GetName().Name}-{scaler.TargetScalerDescriptor.FunctionId}";
```

Comment:
This "GetType().Assembly.GetName().Name" code is repeated 3 times. Recommend defining a private helper for it to ensure we're consistent in the future.